### PR TITLE
Fix multiple log bindings

### LINKED ISSUE #9093: 🐛 Logs print SLF4J warn as errors
_Is this your first time deploying Airbyte:_ no
_OS Version / Instance:_ Linux EC2 m5.2xlarge
_Deployment:_ Docker
_Airbyte Version:_ 0.34.1-alpha
_Source:_ MySQL 0.5.1
_Destination_: Snowflake 0.3.21
_Description:_

```SLF4J
...
destination - 2021-12-18 23:05:00 ERROR () LineGobbler(voidCall):82 - SLF4J: Found binding in [jar:file:/airbyte/lib/log4j-slf4j-impl-2.16.0.jar!/org/slf4j/impl/StaticLoggerBinder.class]
destination - 2021-12-18 23:05:00 ERROR () LineGobbler(voidCall):82 - SLF4J: Found binding in [jar:file:/airbyte/lib/slf4j-log4j12-1.7.25.jar!/org/slf4j/impl/StaticLoggerBinder.class]
```

All of our connections failing now

 https://airbytehq.slack.com/archives/C01MFR03D5W/p1639867371003400?thread_ts=1639867371.003400&cid=C01MFR03D5W

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1202102171868563/1202126595449855) by [Unito](https://www.unito.io)

### LINKED ISSUE #11778: SLF4J: Class path contains multiple SLF4J bindings -MYSQL connector Latest build
Hello Team , 

After Starting latest build and creating connection for mysql , i am getting error of SLF4J: Class path contains multiple SLF4J bindings 
And page reloads to setup source connector page .

┆Issue is synchronized with this [Monday item](https://airbyte-bunch.monday.com/boards/2536787439/pulses/2537097856) by [Unito](https://www.unito.io)

### LINKED ISSUE #13583: Database sources should not log to `stderr` when things are OK
Try `docker run airbyte/source-mysql spec  1>stdour.log 2>stderr.log` and you'll see some warnings emitted on `stderr`:

stdout.log
```
2022-06-07 22:46:39 [32mINFO[m i.a.i.s.p.PostgresSource(main):409 - starting source: class io.airbyte.integrations.source.postgres.PostgresSource
2022-06-07 22:46:39 [32mINFO[m i.a.i.b.IntegrationCliParser(parseOptions):118 - integration args: {spec=null}
2022-06-07 22:46:39 [32mINFO[m i.a.i.b.IntegrationRunner(runInternal):123 - Running integration: io.airbyte.integrations.base.ssh.SshWrappedSource
2022-06-07 22:46:39 [32mINFO[m i.a.i.b.IntegrationRunner(runInternal):124 - Command: SPEC
2022-06-07 22:46:39 [32mINFO[m i.a.i.b.IntegrationRunner(runInternal):125 - Integration config: IntegrationConfig{command=SPEC, configPath='null', catalogPath='null', statePath='null'}

{"type":"SPEC","spec":{"documentationUrl":"https://docs.airbyte.com/integrations/sources/postgres","connectionSpecification":{"$schema":"http://json-schema.org/draft-07/schema#","title":"Postgres Source Spec","type":"object","required":["host","port","database","username"],"additionalProperties":false,"properties":{"host":{"title":"Host","description":"Hostname of the database.","type":"string","order":0},"port":{"title":"Port","description":"Port of the database.","type":"integer","minimum":0,"maximum":65536,"default":5432,"examples":["5432"],"order":1},"database":{"title":"DB Name","description":"Name of the database.","type":"string","order":2},"schemas":{"title":"Schemas","description":"The list of schemas to sync from. Defaults to user. Case sensitive.","type":"array","items":{"type":"string"},"minItems":0,"uniqueItems":true,"default":["public"],"order":3},"username":{"title":"User","description":"Username to use to access the database.","type":"string","order":4},"password":{"title":"Password","description":"Password associated with the username.","type":"string","airbyte_secret":true,"order":5},"jdbc_url_params":{"description":"Additional properties to pass to the JDBC URL string when connecting to the database formatted as 'key=value' pairs separated by the symbol '&'. (example: key1=value1&key2=value2&key3=value3).","title":"JDBC URL Params","type":"string","order":6},"ssl":{"title":"Connect using SSL","description":"Encrypt client/server communications for increased security.","type":"boolean","default":false,"order":7},"replication_method":{"type":"object","title":"Replication Method","description":"Replication method to use for extracting data from the database.","order":8,"oneOf":[{"title":"Standard","additionalProperties":false,"description":"Standard replication requires no setup on the DB side but will not be able to represent deletions incrementally.","required":["method"],"properties":{"method":{"type":"string","const":"Standard","enum":["Standard"],"default":"Standard","order":0}}},{"title":"Logical Replication (CDC)","additionalProperties":false,"description":"Logical replication uses the Postgres write-ahead log (WAL) to detect inserts, updates, and deletes. This needs to be configured on the source database itself. Only available on Postgres 10 and above. Read the <a href=\"https://docs.airbyte.com/integrations/sources/postgres\">Postgres Source</a> docs for more information.","required":["method","replication_slot","publication"],"properties":{"method":{"type":"string","const":"CDC","enum":["CDC"],"default":"CDC","order":0},"plugin":{"type":"string","title":"Plugin","description":"A logical decoding plug-in installed on the PostgreSQL server. `pgoutput` plug-in is used by default.\nIf replication table contains a lot of big jsonb values it is recommended to use `wal2json` plug-in. For more information about `wal2json` plug-in read <a href=\"https://docs.airbyte.com/integrations/sources/postgres\">Postgres Source</a> docs.","enum":["pgoutput","wal2json"],"default":"pgoutput","order":1},"replication_slot":{"type":"string","title":"Replication Slot","description":"A plug-in logical replication slot.","order":2},"publication":{"type":"string","title":"Publication","description":"A Postgres publication used for consuming changes.","order":3}}}]},"tunnel_method":{"type":"object","title":"SSH Tunnel Method","description":"Whether to initiate an SSH tunnel before connecting to the database, and if so, which kind of authentication to use.","oneOf":[{"title":"No Tunnel","required":["tunnel_method"],"properties":{"tunnel_method":{"description":"No ssh tunnel needed to connect to database","type":"string","const":"NO_TUNNEL","order":0}}},{"title":"SSH Key Authentication","required":["tunnel_method","tunnel_host","tunnel_port","tunnel_user","ssh_key"],"properties":{"tunnel_method":{"description":"Connect through a jump server tunnel host using username and ssh key","type":"string","const":"SSH_KEY_AUTH","order":0},"tunnel_host":{"title":"SSH Tunnel Jump Server Host","description":"Hostname of the jump server host that allows inbound ssh tunnel.","type":"string","order":1},"tunnel_port":{"title":"SSH Connection Port","description":"Port on the proxy/jump server that accepts inbound ssh connections.","type":"integer","minimum":0,"maximum":65536,"default":22,"examples":["22"],"order":2},"tunnel_user":{"title":"SSH Login Username","description":"OS-level username for logging into the jump server host.","type":"string","order":3},"ssh_key":{"title":"SSH Private Key","description":"OS-level user account ssh key credentials in RSA PEM format ( created with ssh-keygen -t rsa -m PEM -f myuser_rsa )","type":"string","airbyte_secret":true,"multiline":true,"order":4}}},{"title":"Password Authentication","required":["tunnel_method","tunnel_host","tunnel_port","tunnel_user","tunnel_user_password"],"properties":{"tunnel_method":{"description":"Connect through a jump server tunnel host using username and password authentication","type":"string","const":"SSH_PASSWORD_AUTH","order":0},"tunnel_host":{"title":"SSH Tunnel Jump Server Host","description":"Hostname of the jump server host that allows inbound ssh tunnel.","type":"string","order":1},"tunnel_port":{"title":"SSH Connection Port","description":"Port on the proxy/jump server that accepts inbound ssh connections.","type":"integer","minimum":0,"maximum":65536,"default":22,"examples":["22"],"order":2},"tunnel_user":{"title":"SSH Login Username","description":"OS-level username for logging into the jump server host","type":"string","order":3},"tunnel_user_password":{"title":"Password","description":"OS-level password for logging into the jump server host","type":"string","airbyte_secret":true,"order":4}}}]}}},"supportsNormalization":false,"supportsDBT":false,"supported_destination_sync_modes":[]}}

2022-06-07 22:46:39 [32mINFO[m i.a.i.b.IntegrationRunner(runInternal):171 - Completed integration: io.airbyte.integrations.base.ssh.SshWrappedSource
2022-06-07 22:46:39 [32mINFO[m i.a.i.s.p.PostgresSource(main):411 - completed source: class io.airbyte.integrations.source.postgres.PostgresSource

```

stderr.log
```
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/airbyte/lib/log4j-slf4j-impl-2.17.1.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/airbyte/lib/slf4j-log4j12-1.7.30.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.apache.logging.slf4j.Log4jLoggerFactory]
```

The command worked, and the `stderr` messages are more-or-less meaningless to an end-user. I propose that messages which are not fatal (e.g. causing a crash/excaption/exit) are not emitted on `stderr`.

This was also observed in a `airbyte/source-postgres` as well.  There are likley other connectors that behave the same way.

### PR DESCRIPTION
**Pull request by DoNotPanicUA**
## What
Some imports lead to multiplying log binding. As a result, we have an error in the log.

## How
Explicitly exclude log libs from the imports.

### CODE REVIEW

**Comment by DoNotPanicUA** [comment]
Before:
Screenshot from 2022-07-19 13-49-25
After:
Screenshot from 2022-07-19 13-49-39

**Review by mkhokh-33 — APPROVED** [review]

**Review by subodh1810 — CHANGES_REQUESTED** [review]
@DoNotPanicUA I upgraded mysql to use `airbyte-integrations/bases/debezium-v1-9-2` So you need to make this change there as well. 
Secondly to test this lets run a sync in CDC mode for mysql, postgres and mssql by deploying this change in in local Airbyte instance and see if the logs are as expected or not

**Comment by DoNotPanicUA** [comment]
> @DoNotPanicUA I upgraded mysql to use `airbyte-integrations/bases/debezium-v1-9-2` So you need to make this change there as well. Secondly to test this lets run a sync in CDC mode for mysql, postgres and mssql by deploying this change in in local Airbyte instance and see if the logs are as expected or not

@subodh1810 I've checked `1-9-2` version during implementation. There are no conflicts inside.

MySql tests including CDC : [Error not found](https://github.com/airbytehq/airbyte/runs/7407306688?check_suite_focus=true#step:11:2672) :heavy_check_mark: 
MsSql tests including CDC : [Error not found](https://github.com/airbytehq/airbyte/runs/7466485830?check_suite_focus=true) :heavy_check_mark: 
Postgres tests including CDC : [Error not found](https://github.com/airbytehq/airbyte/runs/7466484424?check_suite_focus=true) :heavy_check_mark: 

I've checked MySql locally on the Airbyte instance and don't see any differences between logs inside destination tests and the connector's run.
Also, I'm going to push only MySql source and destination with this change, as it was requested only there.
Other fixed sources/destinations will be published with other changes.

**Comment by subodh1810** [comment]
@DoNotPanicUA I dont see this change in this file `airbyte-integrations/bases/debezium-v1-9-2/build.gradle`

**Comment by DoNotPanicUA** [comment]
> @DoNotPanicUA I dont see this change in this file `airbyte-integrations/bases/debezium-v1-9-2/build.gradle`

They are not required there.
The `1.9.2` version doesn't have implicit bindings anymore.

**Comment by subodh1810** [comment]
@DoNotPanicUA I pushed a commit to your branch to pull in latest changes from master (I noticed your branch didnt have changes from PR https://github.com/airbytehq/airbyte/pull/14542)

**Comment by DoNotPanicUA** [comment]
@subodh1810 
thanks, I will check the latest state.

**Comment by DoNotPanicUA** [comment]
@subodh1810 
checked mysql. there are no errors as expected.
[logs-26.txt](https://github.com/airbytehq/airbyte/files/9180277/logs-26.txt)

**Review by alexandr-shegeda — APPROVED** [review]

**Comment by subodh1810** [comment]
@DoNotPanicUA we need to run connectors in CDC mode, [here](https://github.com/airbytehq/airbyte/pull/14801#issuecomment-1193865677) the logs are from a non CDC mode sync

**Review by grishick — APPROVED** [review]

**Comment by DoNotPanicUA** [comment]
@subodh1810 
[logs-30.txt](https://github.com/airbytehq/airbyte/files/9197366/logs-30.txt)

Maybe I'm missing something, but why do you expect another behavior of compiled version inside tests where we run it as docker container and inside docker compose where we execute it in the same way?
If there is a difference, shall we change our tests to cover it?

**Comment by DoNotPanicUA** [comment]
/publish connector=connectors/destination-bigquery-denormalized

**Comment by evantahler** [comment]
This PR closes https://github.com/airbytehq/airbyte/issues/13583

**Comment by edgao** [comment]
Is this PR ready to merge? (so that I can publish source-postgres)

**Comment by edgao** [comment]
I'm going to merge this into master as-is (after resolving the merge conflict) - will mark destinations s3 and gcs as unpublished.

**Comment by edgao** [comment]
/publish connector=connectors/destination-bigquery-denormalized

**Review by grishick — APPROVED** [review]

**Comment by DoNotPanicUA** [comment]
@edgao 
Followup issues:
#15285
#15286

**Comment by edgao** [comment]
does destination-databricks need to be published? It has some changes to its build.gradle in this PR also

**Comment by DoNotPanicUA** [comment]
> does destination-databricks need to be published? It has some changes to its build.gradle in this PR also

#15353
The Databricks destination is also broken. [Test run](https://github.com/airbytehq/airbyte/runs/7687769248?check_suite_focus=true)

File: docs/integrations/destinations/bigquery.md
Comment (tuliren) [inline docs/integrations/destinations/bigquery.md:137]: This new version for this PR should be `1.1.13`. `1.1.12` is already allocated to `Fix standard loading mode`. This is fixed in https://github.com/airbytehq/airbyte/pull/15409.

File: docs/integrations/destinations/bigquery.md
Comment (tuliren) [inline docs/integrations/destinations/bigquery.md:180]: Same here, the new version should be `1.1.14`.

**Pull request merged**
